core/src/main/java/io/fabric8/maven/core/config/ResourceConfig.java

```diff
@@ -91,6 +92,8 @@
     @Parameter
     private List<ServiceAccountConfig> serviceAccounts;
 
+    private List<IngressRule> ingressRules;
```

Comment:
@dev-gaur : Why are you using a kuberenetes model class instead of a custom config class used in other xml config resources here?